Source/Plugins/EditorModules/SceneView/Modules/SceneView.cs

```diff
@@ -678,9 +678,16 @@ protected void InitContextMenu()
 							ActionHandler	= this.gameObjectToolStripMenuItem_Click
 						},
 						new MenuModelItem
+						{
+							Name            = typeof(Component).Name,
```

Comment:
Can we add a "..." here to indicate that selecting this menu item will prompt a dialog for user input, rather than immediately performing an action? Would clarify what this item does a bit more by sticking to the "Windows convention".